Yosemite/Yosemite/Model/Payments/StripeAccount+PaymentGatewayAccount.swift

```diff
@@ -0,0 +1,23 @@
+import Hardware
+
+public extension StripeAccount {
```

Comment:
nit: could this be internal?
```suggestion
extension StripeAccount {
```

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5734/commits/0a71bd374e14695fff5e6c1f0716524728893f73